### PR TITLE
ccfri-4050 - NMF bug fixes

### PR DESCRIPTION
**Pull request by vietle-cgi**
# Description

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/RFI/NMF.vue [hunk 1]
```diff
@@ -199,7 +174,6 @@ export default {
     '$route.params.urlGuid': {
       async handler() {
         let ccfriId = this.$route.params.urlGuid;
-        console.log('ccfriId = ', ccfriId);
         await this.loadNmf(ccfriId);
```
Comment (jenbeckett) [inline frontend/src/components/RFI/NMF.vue:177]: This is old code, but do we need to have the variable ccfriId? Or can we just use the urlGuid directly as a parameter to loadNmf?

Comment (vietle-cgi) [reply frontend/src/components/RFI/NMF.vue]: agreed! it's updated

File: frontend/src/components/RFI/NMF.vue [hunk 2]
```diff
@@ -151,19 +125,22 @@
...
+
+import { useAppStore } from '@/store/app.js';
+import { useApplicationStore } from '@/store/application.js';
+import { useNmfAppStore } from '@/store/nmfApp.js';
+import { useNavBarStore } from '@/store/navBar.js';
+
+import alertMixin from '@/mixins/alertMixin.js';
+import rules from '@/utils/rules.js';
 
 let model = { x: [] };
```
Comment (jenbeckett) [inline frontend/src/components/RFI/NMF.vue:140]: Do we need this here? Could we remove this and just declare model in data()?

**Review by jenbeckett — CHANGES_REQUESTED** [review]

File: frontend/src/components/RFI/NMF.vue [hunk 3]
```diff
@@ -173,19 +150,17 @@ export default {
...
   computed: {
-    ...mapState(useApplicationStore, ['formattedProgramYear']),
+    ...mapState(useApplicationStore, ['formattedProgramYear', 'programYearId']),
     ...mapState(useNmfAppStore, ['nmfModel']),
-    ...mapState(useNavBarStore, ['navBarList', 'nextPath', 'previousPath', 'getNavByCCFRIId']),
+    ...mapState(useNavBarStore, ['navBarList', 'nextPath', 'previousPath']),
     ...mapState(useAppStore, ['getFundingUrl']),
     currentFacility() {
-      //return this.getNavByCCFRIId(this.$route.params.urlGuid);
       return this.navBarList.find((el) => el.ccfriApplicationId == this.$route.params.urlGuid);
```
[the remark this answers is not in the document]
Comment (vietle-cgi) [reply frontend/src/components/RFI/NMF.vue]: it's updated

File: frontend/src/components/RFI/NMF.vue (hunk 2 above)
Comment (vietle-cgi) [reply frontend/src/components/RFI/NMF.vue]: it's updated.

File: frontend/src/components/RFI/NMF.vue [hunk 4]
```diff
@@ -151,19 +125,22 @@
...
+import { useNavBarStore } from '@/store/navBar.js';
+
+import alertMixin from '@/mixins/alertMixin.js';
+import rules from '@/utils/rules.js';
 
 let model = { x: [] };
 
 export default {
-  name: 'CcfriRequestMoreInfo',
+  name: 'NmfRequestMoreInfo',
```
Comment (weskubo-cgi) [inline frontend/src/components/RFI/NMF.vue:148]: In general we use the file name as the component name. But if this is consistent with CcfriRequestMoreInfo then it's probably ok. Also this component has a very generic name.

**Review by weskubo-cgi — APPROVED** [review]

**Review by jenbeckett — APPROVED** [review]

**Pull request merged**